src/RestaurantList.jsx

```diff
@@ -0,0 +1,50 @@
+// Here is to call Yelp Fusion API.
+// It temporarily shows a small list of restaurants just to see if the code here calls the API correctly.
+
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import "./RestaurantList.scss";
```

Comment:
this import doesn't work as RestaurantList.scss is not at the same level as this file